test/addons-napi/3_callbacks/binding.c

```diff
@@ -0,0 +1,51 @@
+#include <node_api.h>
+
+void RunCallback(napi_env env, const napi_callback_info info) {
```

Comment:
Fixed in https://github.com/nodejs/node/pull/11975/commits/9c0b1517b894d0a655377d23f9e4b9943d93aa38